README.md

```diff
@@ -1,4 +1,4 @@
-First things first. You're going to need to get a developer key from Careerbuilder. You can obtain a key at http://api.careerbuilder.com/RequestDevKey.aspx.
+First things first. You're going to need to get a developer key from Careerbuilder. To do this, you can reach out to the Partner Team at http://developer.careerbuilder.com/partner_messages/new and request a Developer key.
```

Comment:
I believe it's less "reach out to the partner team" and more "to get a developer key you need to be a partner, to become one, fill out this form".